src/main/java/org/sonarlint/intellij/ui/tree/FlowsTreeModelBuilder.java

```diff
@@ -0,0 +1,76 @@
+/*
+ * SonarLint for IntelliJ IDEA
+ * Copyright (C) 2015 SonarSource
+ * sonarlint@sonarsource.com
+ *
+ * This program is free software; you can redistribute it and/or
+ * modify it under the terms of the GNU Lesser General Public
+ * License as published by the Free Software Foundation; either
+ * version 3 of the License, or (at your option) any later version.
+ *
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the GNU
+ * Lesser General Public License for more details.
+ *
+ * You should have received a copy of the GNU Lesser General Public
+ * License along with this program; if not, write to the Free Software
+ * Foundation, Inc., 51 Franklin Street, Fifth Floor, Boston, MA  02
+ */
+package org.sonarlint.intellij.ui.tree;
+
+import com.intellij.openapi.editor.RangeMarker;
+import java.util.List;
+import javax.annotation.Nullable;
+import javax.swing.tree.DefaultTreeModel;
+import org.sonarlint.intellij.issue.LiveIssue;
+import org.sonarlint.intellij.ui.nodes.LabelNode;
+import org.sonarlint.intellij.ui.nodes.LocationNode;
+import org.sonarlint.intellij.ui.nodes.SummaryNode;
+
+public class FlowsTreeModelBuilder {
+  private SummaryNode summary;
+  private DefaultTreeModel model;
+
+  public FlowsTreeModelBuilder() {
```

Comment:
![CRITICAL](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-critical.png) Add a nested comment explaining why this method is empty, throw an UnsupportedOperationException or complete the implementation. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](https://next.sonarqube.com/sonarqube/coding_rules#rule_key=squid%3AS1186)